### PR TITLE
[DBX-18143] activate 526 state logging

### PR DESCRIPTION
**Pull request by SamStuckey**
## Summary

- *This work is behind a feature toggle (flipper):NO*
- schedule `Form526StateLoggingJob`

## Related issue(s)

- [Ticket](https://github.com/orgs/department-of-veterans-affairs/projects/1263/views/2?filterQuery=assignee%3ASamStuckey&pane=issue&itemId=76015220)
- [Previous work setting up this job](https://github.com/orgs/department-of-veterans-affairs/projects/1263/views/2?filterQuery=assignee%3ASamStuckey&pane=issue&itemId=71313593)
- [how we define success / failure doc](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/526ez/engineering_research/526_scopes.md)

## Testing done

- [ ] *New code is covered by unit tests* N/A
- job has been run in production to hydrate logs, this work just schedules it 

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution (next step)
- [x]  Documentation has been updated (see above)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x]  Feature/bug has a monitor built into Datadog (that's the next step)
- [x]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [x]  I added a screenshot of the developed feature (n/a)

### CODE REVIEW

**Review by freeheeling — APPROVED** [review]
LGTM based on the format of the `AppealsApi::DecisionReviewReportWeekly` job.

File: lib/periodic_jobs.rb [hunk 1]
```diff
@@ -43,6 +43,8 @@
   # Updates status of FormSubmissions per call to Lighthouse Benefits Intake API
   mgr.register('0 2 * * 0', 'Form526StatusPollingJob', 'args' => { paranoid: true })
   # Checks all 'success' type submissions in LH to ensure they haven't changed
+  mgr.register('5 4 * * 7', 'Form526StateLoggingJob')
+  # Log the state of Form 526 submissions to hydrate Datadog monitor
```
Comment (ericboehs) [inline lib/periodic_jobs.rb:47]: The comment descrbing the job should be before the job.

Comment (freeheeling) [reply lib/periodic_jobs.rb:47]: I believe Sam was just following convention in this file, which has, confusingly, included a description **after** the job.

Comment (ericboehs) [reply lib/periodic_jobs.rb:47]: You're right. That's messed up. Haha. I'll make a PR.

**Review by ericboehs — APPROVED** [review]

**Pull request merged**